code/game/atoms.dm

```diff
@@ -649,4 +649,20 @@ its easier to just keep the beam vertical.
 	testing("Set [src]'s gender to [gend], old gender [old_gender] previous gender [prev_gender]")
 
 /atom/proc/mop_act(obj/item/weapon/mop/M, mob/user)
-	return 0
\ No newline at end of file
+	return 0
+
+/atom/proc/get_areaa()
+
+/area/get_areaa()
+	return src

```

Comment:
You do realize I can just do it post haste. You see it's like i'm pseudo testing this with firelocks at live serb, if this is a success then I just replace it.